mod/admin.php

```diff
@@ -253,6 +260,93 @@ function admin_content(App $a) {
 }
 
 /**
+ * @brief Subpage to modify the server wide block list via the admin panel.
+ *
+ * This function generates the subpage of the admin panel to allow the
+ * modification of the node wide block/black list to block entire
+ * remote servers from communication with this node. The page allows
+ * adding, removing and editing of entries from the blocklist.
+ *
+ * @param App $a
+ * @return string
+ */
+function admin_page_blocklist(App $a) {
+	$blocklist = Config::get('system', 'blocklist');
+	$blocklistform = array();
+	if (is_array($blocklist)) {
+		foreach($blocklist as $id =>$b) {
+			$blocklistform[] = Array(
```

Comment:
Standards: Please use lowercase `array` for native variable types. We only use CamelCase for class names.